saos-persistence/src/main/java/pl/edu/icm/saos/persistence/repository/JudgmentRepositoryCustomImpl.java

```diff
@@ -81,8 +78,6 @@ public void delete(List<Long> judgmentIds) {
         
         deleteJudgmentAttributesSql("judgment_lower_court_judgments", judgmentIds);
         
-        deleteEnrichmentTags(judgmentIds);
```

Comment:
Why have you removed deletion of enrichment tags from this? What I meant yesterday what moving the deletion of enrichment tags to a separate repo/service and referring to it from here. If you delete judgment then you don't want to remember that there are two different services: one for removing the judgment alone and second for removing it along with corresponding tags, do you?